bonelab/cli/RapidPrototypeLetters.py

```diff
@@ -0,0 +1,487 @@
+  whiteImage.SetSpacing(el_size_mm)
+  bounds = cube_and_posts.GetOutput().GetBounds()
+  dim = [1,1,1]
+  for i in range(3):
+    dim[i] = (math.ceil((bounds[i*2+1]-bounds[i*2]) / el_size_mm[i]))
+  whiteImage.SetDimensions(dim)
+  whiteImage.SetExtent(0, dim[0] - 1, 0, dim[1] - 1, 0, dim[2] - 1)
+  origin = [1,1,1]
+  origin[0] = bounds[0] + el_size_mm[0] / 2
+  origin[1] = bounds[2] + el_size_mm[1] / 2
+  origin[2] = bounds[4] + el_size_mm[2] / 2
+  whiteImage.SetOrigin(origin)
+  whiteImage.AllocateScalars(vtk.VTK_CHAR,1)
+  for i in range(whiteImage.GetNumberOfPoints()):
+    whiteImage.GetPointData().GetScalars().SetTuple1(i, 127)
+  pol2stenc = vtk.vtkPolyDataToImageStencil()
+  pol2stenc.SetInputData( cube_and_posts.GetOutput() )
+  pol2stenc.SetOutputOrigin( origin )
+  pol2stenc.SetOutputSpacing( el_size_mm )
+  pol2stenc.SetOutputWholeExtent( whiteImage.GetExtent() )
+  pol2stenc.Update()
+  imgstenc = vtk.vtkImageStencil()
+  imgstenc.SetInputData(whiteImage)
+  imgstenc.SetStencilConnection( pol2stenc.GetOutputPort() )
+  imgstenc.ReverseStencilOff()
+  imgstenc.SetBackgroundValue(0)
+  imgstenc.Update()
+  writer = vtkbone.vtkboneAIMWriter()
+  writer.SetInputData( imgstenc.GetOutput() )
+  writer.SetFileName(ofile)
+  writer.Update()
+  
+  exit()
+  
+def CreateShapes(ofile, stl, el_size_mm,
+                 visualize, text, overwrite, height, width):
+  
+  message("Checking if output exists and should overwrite.")
+  if os.path.isfile(ofile) and not overwrite:
+    result = input('File \"{}\" already exists. Overwrite? [y/n]: '.format(ofile))
+    if result.lower() not in ['y', 'yes']:
+      print('Not overwriting. Exiting...')
+      os.sys.exit()
+  
+  message("Create vector text.")
+  vecText = vtk.vtkVectorText()
+  vecText.SetText(text)
+    
+  message("Apply linear extrusion.")
+  extrude = vtk.vtkLinearExtrusionFilter()
+  extrude.SetInputConnection( vecText.GetOutputPort() )
+  extrude.SetExtrusionTypeToNormalExtrusion()
+  extrude.SetVector(0, 0, 1 )
+  extrude.SetScaleFactor (0.5)
+  
+  message("Apply triangle filter.")
+  triangleFilter = vtk.vtkTriangleFilter()
+  triangleFilter.SetInputConnection( extrude.GetOutputPort() )
+  triangleFilter.Update()
+  
+  i_width = triangleFilter.GetOutput().GetBounds()[1] - triangleFilter.GetOutput().GetBounds()[0]
+  i_height = triangleFilter.GetOutput().GetBounds()[3] - triangleFilter.GetOutput().GetBounds()[2]
+  i_depth = triangleFilter.GetOutput().GetBounds()[5] - triangleFilter.GetOutput().GetBounds()[4]
+  message('Input text:',
+          '{:8s} = {:8.2f} mm'.format('width',i_width),
+          '{:8s} = {:8.2f} mm'.format('height',i_height),
+          '{:8s} = {:8.2f} mm'.format('depth',i_depth))
+
+  if (width is not None and height is not None):
+    message('ERROR: User can scale width or height, but not both.',
+            '{:8s} = {:8.2f} mm'.format('width',width),
+            '{:8s} = {:8.2f} mm'.format('height',height))
+    exit()
```

Comment:
Probably `return` here?
```suggestion
    return
```